azkaban-common/build.gradle

```diff
@@ -72,6 +72,13 @@ dependencies {
   testRuntime('com.h2database:h2:1.4.193')
 }
 
+task copySqlFiles(type: Copy) {
+  from project(':azkaban-sql').file('src/sql')
```

Comment:
@kunkun-tang That's what I said earlier. I don't think we need this copy. Why can't we access these files directly ?